Gemfile

```diff
@@ -11,4 +11,5 @@ gem "rest-client", "~> 2.1.0"
 gem "rspec", "~> 3"
 gem "rubocop-govuk", "~> 4.8"
 gem "simplecov", require: false, group: :test
+gem "versionomy", "~> 0.5.0"
```

Comment:
So.. there *is* a specific reason to use versionomy over `Gem::Version` is there?

Reply:
Removed versionomy gem

Reply:
Is `blockenspiel` still needed?

Reply:
It has really gone this time.